packages/medusa/src/services/cart.ts

```diff
@@ -2753,8 +2751,21 @@ class CartService extends TransactionBaseService {
       }
     )
 
+    cart.tax_total = cart.item_tax_total + cart.shipping_tax_total
+
+    cart.raw_discount_total = cart.discount_total
+    cart.discount_total = Math.round(cart.discount_total)
+
+    const giftCardableAmount =
+      (cart.region?.gift_cards_taxable
```

Comment:
question: Should this condition not be reversed?

In case gift cards are taxable, the _giftcardable_ amount would be:
`(subtotal + shipping_total + tax_total) - discount_total`